5.2 Ex/Cup of join.py

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
following Tal's comment, the module's name does not adhere to module names' conventions (a module should not have spaces in its name).

Reply:
I'm sorry, I missed Tal's comment about it.
File's name changed.